symbolic-symcache/tests/test_writer.rs

```diff
@@ -66,7 +52,7 @@ fn test_write_header_linux() -> Result<(), Error> {
         },
         arch: Amd64,
         files: 55,
-        functions: 697,
+        functions: 695,
```

Comment:
`test_write_header_linux` and `test_write_functions_linux` source their data from the same fixture, so I'll include a list of removed functions here.

Here's the two axed functions:
```
ffffffff _ZN15google_breakpad13PageAllocator5AllocEm
ffffffff _ZNK15google_breakpad17LinuxPtraceDumper13BuildProcPathEPciPKc
```
In the snapshot, these have entries with addresses that aren't set to the inline func canary address.